server/src/main/java/io/druid/query/lookup/LookupModule.java

```diff
@@ -119,22 +120,49 @@ public LookupListeningResource(
         })
         {
           @Override
+          public Response handleUpdates(
+              InputStream inputStream, ObjectMapper mapper
+          )
+          {
+            final LookupsState state;
+            try {
+              state = mapper.readValue(inputStream, LookupsState.class);
+            }
+            catch (final IOException ex) {
+              LOG.debug(ex, "Bad request");
+              return Response.status(Response.Status.BAD_REQUEST).entity(ServletResourceUtils.sanitizeException(ex)).build();
+            }
+
+            try {
+              for (Map.Entry<String, LookupExtractorFactoryContainer> e : state.getToLoad().entrySet()) {
+                manager.add(e.getKey(), e.getValue());
+              }
+
+              for (String lookupToDrop : state.getToDrop()) {
+                manager.remove(lookupToDrop);
+              }
+
+              return Response.status(Response.Status.ACCEPTED).entity(manager.getAllLookupsState()).build();
+            }
+            catch (Exception e) {
+              LOG.error(e, "Error handling request");
+              return Response.serverError().entity(ServletResourceUtils.sanitizeException(e)).build();
+            }
+          }
+
+          @Override
           public Object post(final Map<String, LookupExtractorFactory> lookups)
               throws Exception
           {
             final Map<String, LookupExtractorFactory> failedUpdates = new HashMap<>();
             for (final String name : lookups.keySet()) {
-              final LookupExtractorFactory factory = lookups.get(name);
-              try {
-                // Only fail if it should have updated but didn't.
-                if (!manager.updateIfNew(name, factory) && factory.replaces(manager.get(name))) {
-                  failedUpdates.put(name, factory);
```

Comment:
I've removed the lookup node APIs that do the update from docs.... users only need to know about "get" apis for debugging
in no case, should user manually load lookups on the nodes